app/docker/networks/edit/NetworkDetailsTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { Fragment } from 'react';
+import DockerNetworkHelper from 'Docker/helpers/networkHelper';
+
+import { Widget, WidgetBody, WidgetTitle } from '@/portainer/components/widget';
+import { DetailsTable } from '@/portainer/components/DetailsTable';
+import { Button } from '@/portainer/components/Button';
+import { Authorized } from '@/portainer/hooks/useUser';
+
+import { isSystemNetwork } from '../network.helper';
+import { DockerNetwork, IPConfig } from '../types';
+
+interface Props {
+  network: DockerNetwork;
+  onRemoveNetworkClicked: () => void;
+}
+
+export function NetworkDetailsTable({
+  network,
+  onRemoveNetworkClicked,
+}: Props) {
+  const allowRemoveNetwork = !isSystemNetwork(network.Name);
+  const IPV4Configs: IPConfig[] = DockerNetworkHelper.getIPV4Configs(
```

Comment:
use lowercase camelCase for variable names
```suggestion
  const ipv4Configs: IPConfig[] = DockerNetworkHelper.getIPV4Configs(
```